dev-python/influxdb/influxdb-5.3.0-r1.ebuild

```diff
@@ -30,6 +30,8 @@ BDEPEND="test? (
 		dev-python/requests-mock[${PYTHON_USEDEP}]
```

Comment:
With distutils_enable_tests, there's no need to put ${RDEPEND} in test? or to add nose as a testdep. It's very cconvenient!

Reply:
Done.